raiden/utils/cli.py

```diff
@@ -370,11 +371,20 @@ def fixed_gas_price_strategy(_web3, _transaction_params):
                 # the convert() method has to be idempotent, so when the value is callable
                 # we assume it is already the correct gas-price-strategies function
                 return value
+
+            gas_strategies = {
+                "normal": fast_gas_price_strategy,
+                "fast": faster_gas_price_strategy,
+                "rpc": rpc_gas_price_strategy,
+            }
+
             gas_price_string = super().convert(value, param, ctx)
-            if gas_price_string == "fast":
-                return faster_gas_price_strategy
+            
+            if gas_price_string not in gas_strategies:
+                raise ConfigurationError(f"Unknown gas price (strategy): {gas_price_string}")
+
             else:
-                return fast_gas_price_strategy
+                return gas_strategies[gas_price_string]
```

Comment:
Looking better, but this `else` is redundant now, we could do without it.

Reply:
I kinda like a good if-else block. But I don't have a too strong opinion for such a case here. So I'll follow your recommendation.

Reply:
Same commiting-strategy as [explained above](https://github.com/raiden-network/raiden/pull/7510#discussion_r816673545)

Reply:
✔️